### DISCUSSION TITLE
Images / Metags for content good SEO

### DISCUSSION THREAD

**Discussion by Rar9** [directus/directus#15582] [Archive]
If directus is used as a cms backend to provide headless data, as simple guide might help to avoid basic setup mistakes.

Example, if images are created nowadays 
Alternative text & Titels as well as images dimensions are required.  This also means responsive images handling with sharp should be touched on as well if required. 

For  metatags, title and description are a minimum must have. 
To actually succeed opengraph /twitter and other social schemas are required. 

These images /metatags basics every website user will need to create at some point, if a directus is used for website backend. 

So why is there no plugging to quickly add these nesseary fields or at least provide a documation to get theses add it to directus the correct way to be able to use with each collection?

**Comment by rijkvanzanten** [comment]
> So why is there no plugging to quickly add these nesseary fields or at least provide a documation to get theses add it to directus the correct way to be able to use with each collection?

There's already a couple default fields under directus_files that you can use for this purpose on a global level. If you need per-item overrides for the image metadata, you can just add more fields to your collection 🙂 

As for auto-generating the correct sizes with sharp, sharp is available through the API's `/assets` endpoint: https://docs.directus.io/reference/files.html#requesting-a-thumbnail
